nifi-commons/nifi-property-encryptor/src/test/java/org/apache/nifi/encrypt/KeyedCipherPropertyEncryptorTest.java

```diff
@@ -77,26 +70,26 @@ public void testEncryptHexadecimalEncoded() throws DecoderException {
     @Test
     public void testDecryptEncryptionException() {
         final String encodedProperty = Hex.encodeHexString(PROPERTY.getBytes(DEFAULT_CHARSET));
-        assertThrows(EncryptionException.class, () -> encryptor.decrypt(encodedProperty));
+        assertThrows(Exception.class, () -> encryptor.decrypt(encodedProperty));
```

Comment:
Why not retain the more specific exception type here?

Reply:
The thrown exception is different based the Java version, so switching to the more generic Exception provided the least amount of impact to the unit test.